keras2onnx/ke2onnx/layer_spec.py

```diff
@@ -28,8 +28,11 @@ def _simple_layer_name_extractor(fstr_list, node_name):
 
 def _conv_layer_spec_outputs(layer, node):
     if type(layer) == _layer.DepthwiseConv2D:
-        ri = node.name.rindex('/')
-        return node.name[:ri + 1] + 'BiasAdd'
+        if node.name.endswith('depthwise'):
```

Comment:
Can you check if there are any following node with the same layer?

Reply:
I see an op called `efficientnet-b0/block1a_dwconv/depthwise/(depthwise)`